compiler/optimizer/Dominators.hpp

```diff
@@ -55,30 +54,26 @@ class TR_Dominators
    int             dominates(TR::Block *block, TR::Block *other);
 
    TR::Compilation * comp()         { return _compilation; }
-   TR_Memory *      trMemory()      { return comp()->trMemory(); }
-   TR_StackMemory   trStackMemory() { return trMemory(); }
    bool trace() { return _trace; }
 
-   TR::deque<int32_t> _dfNumbers;
-
    protected:
 
-   typedef TR::SparseBitVector SparseBitVector;
+   typedef TR_BitVector BitVector;
 
    private:
 
    struct BBInfo
       {
-      BBInfo(TR::Allocator allocator) :
-         _bucket(allocator), _block(NULL), _parent(-1), _idom(-1), _ancestor(-1), _label(-1), _child(-1), _sdno(-1), _size(0)
+      BBInfo(TR::Region &region) :
```

Comment:
I clearly missed it myself when going through this, but it might make sense to make this constructor explicit and explicitly construct the BBInfo from from a region in locations where this currently happens automagically.